staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/register.go

```diff
@@ -42,10 +42,23 @@ func AddToGroupVersion(scheme *runtime.Scheme, groupVersion schema.GroupVersion)
 		schema.GroupVersion{Group: groupVersion.Group, Version: runtime.APIVersionInternal}.WithKind(WatchEventKind),
 		&InternalEvent{},
 	)
+	scheme.AddKnownTypes(groupVersion, &ListOptions{})
 	scheme.AddConversionFuncs(
 		Convert_versioned_Event_to_watch_Event,
 		Convert_versioned_InternalEvent_to_versioned_Event,
 		Convert_watch_Event_to_versioned_Event,
 		Convert_versioned_Event_to_versioned_InternalEvent,
 	)
 }
+
+// scheme is the registry for the common types that adhere to the meta v1 API spec.
+var scheme = runtime.NewScheme()
```

Comment:
We will eventually be adding things that are not parameters, things like the structured responses for get / describe and probably List.